packages/system/src/border.test.ts

```diff
@@ -1,16 +1,7 @@
 import { border, BorderProps } from "./border";
-import { beforeAll, describe, expect, test } from "vitest";
-import { theme } from "@kuma-ui/sheet";
+import { describe, expect, test } from "vitest";
 
 describe("border utility function", () => {
-  beforeAll(() => {
-    theme.setUserTheme({
-      radii: {
-        sm: "1rem",
-      },
-    });
-  });
```

Comment:
Is it possible to keep the theme test instead of removing it?

Reply:
@kotarella1110 
These processes would actually take place after the user theme has already been converted.
Therefore, I believe that testing the theme is unnecessary for this test

Reply:
@nojiritakeshi You are right. I think this line can be deleted.
However, you should keep the following test case!

https://github.com/kuma-ui/kuma-ui/pull/292/files#diff-a474cb7251af443b69bc46e96a3ce239067c318dd9f38a86cca45ef2b8020c6cL78

Reply:
@kotarella1110 
Is it correct that the code that should be left behind is the following?
`[{ borderRadius: "sm" }, "border-radius: 1rem;", ""],`

I think that the value passed as "sm" cannot be 1rem at the time this process is performed.

Reply:
@nojiritakeshi It means that I want to keep the test case of borderRadius itself as follows.

```
[{ borderRadius: "1rem" }, "border-radius: 1rem;", ""],
```